src/schemas/json/smapi-manifest-3.17.0.json

```
@@ -0,0 +1,89 @@
+{
+  "$comment": "https://stardewvalleywiki.com/Modding:Modder_Guide/APIs/Manifest",
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "type": "object",
+  "additionalProperties": false,
+  "definitions": {
+    "semver": {
+      "type": "string",
+      "description": "Semantic version number (e.g. 1.2.3-alpha.4).",
+      "pattern": "^(\\d|[1-9]\\d*)\\.(\\d|[1-9]\\d*)\\.(\\d|[1-9]\\d*)(-(alpha|beta|rc)(\\.(\\d|[1-9]\\d*))?)?$"
+    },
+    "slug": {
+      "type": "string",
+      "description": "Slug type (alphanumeric and periods).",
+      "pattern": "^[\\.\\w\\d]+$"
+    },
+    "dependency": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "UniqueID": {
+          "description": "The unique ID of the mod this content pack is for.",
+          "$ref": "#/definitions/slug"
+        },
+        "MinimumVersion": {
+          "description": "The minimum version of the mod this content pack is for. If the player doesn't have the minimum version, SMAPI will show an error message.",
+          "$ref": "#/definitions/semver"
+        }
+      }
+    },
+    "dllFile": {
+      "type": "string",
+      "description": "Relative path to a DLL file.",
+      "pattern": "^.+\\.dll$"
+    }
+  },
+  "description": "Manifest for a SMAPI mod.",
+  "properties": {
+    "Name": {
+      "type": "string",
+      "description": "The mod name. SMAPI uses this in player messages, logs, and errors.",
+      "minLength": 1
+    },
+    "Author": {
+      "type": "string",
+      "description": "The name of the person who created the mod. Ideally this should include the username used to publish mods.",
+      "minLength": 1
+    },
+    "Version": {
+      "description": "The mod's semantic version. Make sure you update this for each release! SMAPI uses this for update checks, mod dependencies, and compatibility blacklists (if the mod breaks in a future version of the game).",
+      "$ref": "#/definitions/semver"
+    },
+    "Description": {
+      "type": "string",
+      "description": "A short explanation of what your mod does (one or two sentences), shown in the SMAPI log.",
+      "minLength": 1
+    },
+    "UniqueID": {
+      "description": "A unique identifier for your mod. The recommended format is <your name>.<mod name>, with no spaces or special characters.",
+      "$ref": "#/definitions/slug"
+    },
+    "EntryDll": {
+      "description": "The mod's compiled DLL filename in its mod folder.",
+      "$ref": "#/definitions/dllFile"
+    },
+    "ContentPackFor": {
+      "description": "The unique ID of the mod this content pack is for.",
```

```suggestion
      "description": "An ID of the current mod content pack is for\nhttps://stardewvalleywiki.com/Modding:Modder_Guide/APIs/Manifest#Fields,
```

This issue is the same as above.